### PR TITLE
Project_1 Week_2: Review project from trials

### PR DESCRIPTION
**Pull request by Leon-Espitia**
In this project I:

- Copy the code from one of the Microverse trials.
- Provide my own review.

### CODE REVIEW

**Review by Leon-Espitia — COMMENTED** [review]
#Changes required 🧑‍🔧 ♻️ 
Please check my comments below and make all required changes.

In addition ➕ 
*Add the Readme File and please Run Linters.


Submit for a re-review when you are done.

Thank´s. 😃

File: index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      <img
+        class="project-tile"
+        src=""
+        alt="project"
+      />
+      <p class="project-title">
+        <span class="code">&lt;</span>
+        Hold a project 1
+        <span class="code">&#47;&gt;</span>
+      </p>
```
[the remark this answers is not in the document]
Comment (Chimwemwe-127001) [reply index.html:63]: - [ ] I suggest you include an explanation, a "why", for each requested change. In addition, I recommend adding a link to back up your explanation. This approach gives the student the actionable guidance required to complete the project.

**Review by Chimwemwe-127001 — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
### Hi @Leon-Espitiahi

You've done an amazing job with the code review so far :fire: :tada: 

There are some issues that you still need to work on to go to the next project but you are almost there! :muscle:

## Highlights :+1: 
:heavy_check_mark: Good utilization of GitHub inline comments feature in the review
:heavy_check_mark: Great job spotting the lack of semantic tags

## A few things to always keep in mind ♻️
- Your comments should always be phrased to sound more suggestive.
- Always include an explanation, a "why", for each requested change 
- Make sure that your comments are clear and easy to understand
- Use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖

Check the comments under the review to find which comments can be altered.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cwaku — APPROVED** [review]
Hi @Leon-Espitia 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good utilization of GitHub inline comments feature in the review
 - [x]  Great job spotting the lack of semantic tags
 - [x] Use of emojis to make the code review more human-friendly 👽🤖.
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**